extensions/vi-mode/options.lisp

```diff
@@ -376,6 +379,37 @@
                    (option-value option))
             :test 'equal)))))
 
+(defvar *default-non-broad-word-char* (mapcar #'string '(#\Newline #\Space #\Tab)))
+
+(defun compile-non-broad-word-char (value)
+  (compile-rules value "non-broad-word-char"))
+
+(define-option "non-broad-word-char" 
```

Comment:
Then I suggest `isWORDseparator` because in Vim broad word is called WORD. Otherwise it is not clear separator of what it is.

Reply:
Even better. That sounds reasonable in some sense, but the upcased name is also strange since all Vim's options have downcased names.
Vim has `isprint` to set custom printable chars, so `isseparator` doesn't look weird to me, though.

Reply:
OK. Let it be `isseparator` then.